wsgi/odyssy/announcement/models.py

```diff
@@ -1,18 +1,24 @@
-from __future__ import unicode_literals
-
-import basic
+from basic import utils
 from datetime import datetime, time
 
 from django.db import models
 from django.core.exceptions import ValidationError
-from django.db.models import Q
+
+
+def get_current_date():
+    """ set default as start of date """
+    return datetime.combine(datetime.today(), time.min)
+
+def get_final_date():
+    """ set end as undefined date """
+    return datetime.combine(datetime.max, time.max)
```

Comment:
Make these methods as `get_today_start` and `get_today_end` and move them to `utils` too. Make changes wherever needed.